org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -358,7 +359,8 @@ private Panel newGadgetPanel(MDashboardPreference dp, MDashboardContent dc) {
 	}
 	
 	private void renderHelpButton(Caption caption, String text) {
-		ToolBarButton help = new ToolBarButton();
+		A help = new A();
```

Comment:
since you just need mouse over event, shouldn't a label works just fine here ? An A element is still a clickable element and that's not desirable here (you might have issue with :active and :visited state too)

Reply:
If you mean the org.adempiere.webui.component.Label class - it does not have the setIconSclass() method. Would you suggest to implement the setIconSclass() and setImage() methods, or extend the LabelImageElement abstract class from Label?

Not sure if, and how would these work, do you have any suggestions?

Reply:
ah ok, my mistake, label wouldn't work here.